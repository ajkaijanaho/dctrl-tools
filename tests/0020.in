Depends: foo, libprce3-dev, bar

Depends: foo, libprce3, bar

