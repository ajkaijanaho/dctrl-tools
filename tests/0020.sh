#!/bin/sh

set -e

LC_ALL=C
export LC_ALL

$GREP_DCTRL -FDepends -w libprce3


