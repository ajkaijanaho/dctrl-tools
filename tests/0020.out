Depends: foo, libprce3, bar

